app/Transformers/PlanTransformerBase.php

```diff
@@ -59,6 +59,8 @@ public function parseTranslationDataFileset(?BibleFileset $fileset) : Array
             "bitrate" => $fileset->bitrate,
             "codec" => $fileset->codec,
             "container" => $fileset->container,
+            // December 2022, this is temporary to support older versions of 5fish applications. Revisit in one year after discussing with James Thompson
```

Comment:
From the filename, this appears to be part of a Plan, which is not part of the public API, son 5fish would not have access. please remove this

Reply:
@bradflood , ok I got it, I just removed it.